More info on this patch:

  http://mail.gnu.org/archive/html/libtool/2003-09/msg00060.html
  http://mail.gnu.org/archive/html/libtool/2003-09/msg00065.html

This patch has been sent for merge upstream in libtool.


Index: libltdl/ltdl.c
===================================================================
RCS file: /cvs/kaffe/kaffe/libltdl/ltdl.c,v
retrieving revision 1.35
diff -u -r1.35 ltdl.c
--- libltdl/ltdl.c	4 Jun 2003 16:03:50 -0000	1.35
+++ libltdl/ltdl.c	22 Sep 2003 13:33:23 -0000
@@ -39,6 +39,10 @@
 
 #if HAVE_STDLIB_H
 #  include <stdlib.h>
+#else
+#  if HAVE_MALLOC_H
+#    include <malloc.h>
+#  endif
 #endif
 
 #if HAVE_STRING_H
@@ -51,10 +55,6 @@
 
 #if HAVE_CTYPE_H
 #  include <ctype.h>
-#endif
-
-#if HAVE_MALLOC_H
-#  include <malloc.h>
 #endif
 
 #if HAVE_MEMORY_H
